interpreter/terminal_interface/magic_commands.py

```diff
@@ -38,8 +40,9 @@ def handle_help(self, arguments):
       "%debug [true/false]": "Toggle debug mode. Without arguments or with 'true', it enters debug mode. With 'false', it exits debug mode.",
       "%reset": "Resets the current session.",
       "%undo": "Remove previous messages and its response from the message history.",
-      "%save_message [path]": "Saves messages to a specified JSON path. If no path is provided, it defaults to 'messages.json'.",
-      "%load_message [path]": "Loads messages from a specified JSON path. If no path is provided, it defaults to 'messages.json'.",
+      "%save_message [path]": "Saves messages to a specified JSON path. If no path is provided, it defaults to '[date]_messages_[counter].json'.",
+      "%load_message [path]": "Loads messages from a specified JSON path. If no path is provided, it loads the latest default message '[date]_messages_[counter].json'.",
+      "%delete_message [number]": "Deletes saved messages, preserving the most recent 'number' files. If no number is provided, it deletes all but the most recent file.",
```

Comment:
`%delete_message` didn't seem to work. Maybe because it's defined as `delete_messages` on [line 148 in the `handle_magic_command()` method](https://github.com/KillianLucas/open-interpreter/pull/566/files#diff-39f7ea35aad6a3d890783709177df802705530ae6f13d67a43a94656d7b15076R148)?

![Screen Shot 2023-10-08 at 12 02 04 PM](https://github.com/KillianLucas/open-interpreter/assets/1667415/9b12730a-d344-4420-aadf-b2f79f29148e)

Also, it seems like `%delete_message` should require a path rather than potentially deleting multiple files when called without an argument.

Reply:
Yeah, it should be %delete_messages, and will make it so that you can supply a folder or file.